packages/tiktok-video-element/tiktok-video-element.js

```diff
@@ -0,0 +1,370 @@
+// https://developers.tiktok.com/doc/embed-player
+
+const EMBED_BASE = 'https://www.tiktok.com/player/v1';
+const MATCH_SRC = /tiktok\.com\/(?:@[^/]+\/video\/)?(\d+)(?:\/([\w-]+))?/;
```

Comment:
@ronalduQualabs made some updates here. See [`21c0ae4` (#145)](https://github.com/muxinc/media-elements/pull/145/commits/21c0ae4ba9916c5bfffcc5d5bf60e3b37c812981)

Primarily changes related to styling, it's better to keep things consistent between media elements for easier maintenance later. 

Removed the resolve id and support of using an id as src because that's how the media elements have used the src historically. Resolving many urls like short urls etc is cool but not needed (yet). 

Removed the `config` attribute, we don't want to support that for the time being.

FYI `false` attributes are the absence of that attribute in HTML. We shouldn't have to support someone using attributes wrong and adding a string `false` value.